### PR TITLE
Replace app link with video link

### PR DESCRIPTION
**Pull request by goyalyashpal**
... took from README file

### CODE REVIEW

**Comment by flide** [comment]
Can't approve this change.
We can't just say - "here is a link for the game" and post a video link.
Although I understand the link does not work anymore, I am sure there can be a better solution.

**Comment by goyalyashpal** [comment]
okay, i will see if smth can be done

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Usability Guide
 
-So, what capabilities does 8Vim have? Once you know how to type with this thing (Learn how to type with this original [8Pen-game App] (https://play.google.com/store/apps/details?id=com.eightpen.android.wordcup&hl=en), you must know the following
+So, what capabilities does 8Vim have? Once you know how to type with this thing (Learn how to type with this original [8Pen-game video](https://www.youtube.com/watch?v=99vsUF4NuLk), you must know the following
```
Comment (goyalyashpal) [inline fastlane/metadata/android/en-US/full_description.txt:5]: 1. rechecking the whole contents:
    * i think you have missed that I have not changed just the link, but wording as well. 
    * It doesnt just says `[game](video)`, it says `[game video](video)`
    * ... and it seems pretty consistent to me, there's no mention of game/app elsewhere in this whole file

Some very slight improvements can be:

2. Being verbose, so:
    * `PR:` "8Pen-game video"
    *  `A :` "8Pen-game's video"
    * ` B :` "8Pen-game intro video"

3. "with" --> "from"

```diff
- Learn ... with this ... video ...
+ Learn ... from this ... video ...
```

4. Can u please recheck regarding 1st point (game vs video) ?? And also share your preference/opinion on second point (verbose) ?

**Comment by goyalyashpal** [comment]
wow, this pr is one year old. so much have happened since then haha... please ignore 😅

**Pull request merged**